ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/activityBlock/activityBlock.component.html

```diff
@@ -1,8 +1,9 @@
 <div class="activity-block-container">
     <div *ngIf="block.title" [innerHTML]="block.title"></div>
 
-    <div *ngIf="childInstances?.length" class="cards-block">
-        <div *ngFor="let instance of childInstances; trackBy: getInstanceId" class="card">
+    <div *ngIf="childInstances?.length" class="child-instances">
+        <div *ngFor="let instance of childInstances; trackBy: getInstanceId"
+             class="instance" [ngClass]="isModal ? 'modal' : 'embedded'">
```

Comment:
**#minor** wondering if we can do [ngClass]="['instance', isModal ? 'modal' : 'embedded']"

Reply:
Seems to me to be pretty the same, isn't it ?

Reply:
true, pretty the same, that's why I marked the comment as minor :) just using one attribute for class might be considered as an alternative

Reply:
My point of view on that: 
- use `class` for "constant" classes,
- use `ngClass` for classes which are dependent on some conditions.

Reply:
yea, no problem, I guess it's just an old habit since there were some issues when an element has both class and ngClass but it should have been fixed already

Reply:
yeah, I'm an old hobbit and have old habits